dags/atd_service_bot_index_issues_to_dts_portal.py

```diff
@@ -0,0 +1,66 @@
+import os
+from datetime import datetime, timedelta
+
+from airflow.models import DAG
+from airflow.operators.docker_operator import DockerOperator
+
+from onepasswordconnectsdk.client import Client, new_client
+import onepasswordconnectsdk
+
+DEPLOYMENT_ENVIRONMENT = os.getenv("ENVIRONMENT", "development")
+ONEPASSWORD_CONNECT_HOST = os.getenv("OP_CONNECT")
+ONEPASSWORD_CONNECT_TOKEN = os.getenv("OP_API_TOKEN")
+VAULT_ID = os.getenv("OP_VAULT_ID")
+
+default_args = {
+    "owner": "airflow",
+    "description": "Create/update 'Index' issues in the DTS portal from Github.",
+    "depends_on_past": False,
+    "start_date": datetime(2015, 12, 1),
+    "email_on_failure": False,
+    "email_on_retry": False,
+    "retries": 0,
+}
+
+docker_image = "atddocker/atd-service-bot:production"
+
+REQUIRED_SECRETS = {
+    "KNACK_APP_ID": {
+        "opitem": "Service Bot",
+        "opfield": f"{DEPLOYMENT_ENVIRONMENT}.knackAppId",
+        "opvault": VAULT_ID,
+    },
+    "KNACK_API_KEY": {
+        "opitem": "Service Bot",
+        "opfield": f"{DEPLOYMENT_ENVIRONMENT}.knackApiKey",
+        "opvault": VAULT_ID,
+    },
+    "GITHUB_ACCESS_TOKEN": {
+        "opitem": "Service Bot",
+        "opfield": "shared.githubAccessToken",
+        "opvault": VAULT_ID,
+    },
+}
+
+client: Client = new_client(ONEPASSWORD_CONNECT_HOST, ONEPASSWORD_CONNECT_TOKEN)
+env_vars = onepasswordconnectsdk.load_dict(client, REQUIRED_SECRETS)
+
+with DAG(
+    dag_id=f"atd_service_bot_issues_to_dts_portal_{DEPLOYMENT_ENVIRONMENT}",
+    default_args=default_args,
+    schedule_interval="13 7 * * *",
+    dagrun_timeout=timedelta(minutes=60),
+    tags=["repo:atd-service-bot", "knack", "github"],
+    catchup=False,
+) as dag:
+    t1 = DockerOperator(
```

Comment:
You mentioned that you had not refactored this to use the `@dag()` and `@task()` operators. I think the way you've done this is fantastic - there is no real upside to using the decorators as opposed to doing it this way. You might argue that using them can lead to shorter DAGs, but when you're porting an airflow-1 DAG to airflow-2, I think going with the "classic" syntax is great. Airflow 2 supports both fully.